float64 rotation_angle_z
---
